ts/src/okx.ts

```diff
@@ -1632,7 +1632,7 @@ export default class okx extends Exchange {
             'contractSize': contract ? this.safeNumber (market, 'ctVal') : undefined,
             'expiry': expiry,
             'expiryDatetime': this.iso8601 (expiry),
-            'strike': strikePrice,
+            'strike': this.parseNumber (strikePrice),
```

Comment:
@ttodua funnily 😅 
![image](https://github.com/user-attachments/assets/1c19dc64-2dac-4198-82fe-0226b4030254)

Reply:
heh ;)

Reply:
I am going through the issues that break builds, strange havent noticed it earlier